standalone/content/js/webapp.jsx

```diff
@@ -204,6 +204,9 @@ loop.webapp = (function(_, OT, mozL10n) {
     var participantStore = new loop.store.ParticipantStore(dispatcher);
     var serverConnectionStore = new loop.store.ServerConnectionStore(dispatcher, {});
 
+    dispatcher.dispatch(
+      new sharedActions.SetOwnDisplayName({ displayName: "Bill Smith" }));
```

Comment:
Maybe to spice things up... `"Bill Smith".split("").sort(() => Math.random() - .5).join("")`